File: 2698-find-the-array-concatenation-value/2698-find-the-array-concatenation-value.py
class Solution:
    def findTheArrayConcVal(self, nums: List[int]) -> int:
        i = 0
        r = len(nums)-1
        str1 = ''
        num = 0
        while i<=r:
            if len(nums) == 1:
                num = int(nums[i]) + num
                return num
            str1 = str(nums[i])+str(nums[r])
            num = int(str1)+num
            nums.pop(i)
            nums.pop(r-1)
            r-=2
        return num

File: 2698-find-the-array-concatenation-value/2698-find-the-array-concatenation-value.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findTheArrayConcVal = function (nums) {
    let i = 0
    let r = nums.length - 1
    let str = ''
    let num = 0
    while (i <= r) {
        if (nums.length == 1) {
            num = Number(nums[i]) + num
            return num
        }
        str = (nums[i]).toString() + (nums[r]).toString()
        num = Number(str) + num
        nums.splice(i, 1)
        nums.splice(r - 1, 1)
        r = r - 2
    }
    return num
};